crates/control/src/time_to_reach_kick_position.rs

```diff
@@ -90,27 +79,17 @@ impl TimeToReachKickPosition {
                 .mul_f32(angle * FRAC_1_PI)
         });
 
-        let time_to_reach_kick_position = walk_time.map(|walk_time| {
-            [
-                walk_time,
-                *context
-                    .stand_up_back_estimated_remaining_duration
-                    .unwrap_or(&Duration::ZERO),
-                *context
-                    .stand_up_front_estimated_remaining_duration
-                    .unwrap_or(&Duration::ZERO),
-                time_to_turn,
-            ]
-            .into_iter()
-            .fold(Duration::ZERO, Duration::saturating_add)
-        });
-
-        context.time_to_turn.fill_if_subscribed(|| time_to_turn);
-        context
-            .time_to_reach_kick_position_output
-            .fill_if_subscribed(|| time_to_reach_kick_position);
-        *context.time_to_reach_kick_position = time_to_reach_kick_position.unwrap_or(Duration::MAX);
+        let time_to_reach_kick_position = [
+            walk_time,
+            *context.stand_up_back_estimated_remaining_duration,
+            *context.stand_up_front_estimated_remaining_duration,
+            time_to_turn,
+        ]
+        .into_iter()
+        .fold(Duration::ZERO, Duration::saturating_add);
```

Comment:
Why is this a fold and not a sum? Is the saturation property actually so important? Do we run into overflows?

Reply:
The remaining duration of the standup motions may be `Duration::MAX` if the motion was aborted.
This happens for example when the robot falls down again during standup.

Adding anything to this value results in an overflow panic.